src/wagtail_live/utils.py

```diff
@@ -0,0 +1,32 @@
+"""Wagtail Live utils"""
+
+from importlib import import_module
+
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+
+from wagtail_live.receivers import BaseMessageReceiver
+
+
+def get_live_receiver():
+    """Retrieves the live receiver chosen.
+
+    Returns:
+        (BaseMessageReceiver) corresponding to the live receiver specified if found else None.
+    """
+
+    live_receiver = getattr(settings, "WAGTAIL_LIVE_RECEIVER", "")
+    if not live_receiver:
+        # Assume live interface is used, in which case no additional setup is needed.
+        return
+
+    dotted_path, receiver_name = live_receiver.rsplit(".", 1)
+    module = import_module(dotted_path)
+    receiver = getattr(module, receiver_name)
+
+    if not issubclass(receiver, BaseMessageReceiver):
+        raise ImproperlyConfigured(
+            "The receiver specified doesn't inherit from "
```

Comment:
I don't understand, like "The receiver receiver-chosen isn't supported" ?

Reply:
I meant that it would show a message like this:
> The receiver myapp.receivers.MyFaultyReciver doesn't inherit from wagtail_live.receivers.BaseMessageReceiver

Reply:
Ah ok, yes it's more informative indeed.

Reply:
Done!